site/docs/main/configuration.md

```diff
@@ -189,6 +190,34 @@ If present, the value of the `CONTOUR_NAMESPACE` environment variable is used as
 
 The `CONTOUR_NAMESPACE` environment variable is set via the [Downward API][6] in the Contour [example manifests][7].
 
+## Bootstrap Config File
+
+The bootstrap configuration file is generated by an initContainer in the Envoy daemonset which runs the `contour bootstrap` command to generate the file.
+This configuration file configures the Envoy container to connect to Contour and receive configuration via xDS.
+
+The next section outlines all the available flags that can be passed to the `contour bootstrap` command which are used to customize
+the configuration file to match the environment in which Envoy is deployed. 
+
+### Flags
+
+There are flags that can be passed to `contour boostrap` that help configure how Envoy
+connects to Contour:
+
+| Flag | Default  | Description |
+|------------|----------|-------------|
+| <nobr>--resources-dir</nobr> | "" | Defines the directory where resource files will be written.  |
+| <nobr>--admin-address</nobr> | 127.0.0.1 | Defines the address the Envoy admin webpage will listen on.  |
+| <nobr>--admin-port</nobr> | 9001 | Defines the port the Envoy admin webpage will listen on.  |
+| <nobr>--xds-address</nobr> | 127.0.0.1 | Defines the address to connect to Contour xDS server on.  |
+| <nobr>--xds-port</nobr> | 8001 | Defines the port to connect to Contour xDS server on. |
+| <nobr>--envoy-cafile</nobr> | "" | Defines the gRPC CA filename for Envoy to load.  |
+| <nobr>--envoy-cert-file</nobr> | "" | Defines the gRPC client cert filename for Envoy to load.  |
+| <nobr>--envoy-key-file</nobr> | "" | Defines the gRPC client key filename for Envoy to load.  |
+| <nobr>--namespace</nobr> | projectcontour | Defines the namespace the Envoy container will run, also configured via ENV variable "CONTOUR_NAMESPACE". Namespace is used as part of the metric names on static resources defined in the boostrap configuration file.    |
+| <nobr>--xds-server-version</nobr> | v2 | This field specifies the xDS API version to use. Options are `v2` or `v3`.  |
```

Comment:
name of the flag needs to be updated to `--xds-resource-version` now.

Reply:
This is removed in #3084

Reply:
this is the bootstrap flag though: https://github.com/projectcontour/contour/blob/main/cmd/contour/bootstrap.go#L37

Reply:
🤯

Reply:
Ok so I did do the right thing in #3084 and left this here. =)

I just pushed and updated this to `--xds-resource-version`.